tools.py

```diff
@@ -112,8 +112,8 @@ def git_add_and_pr(self, gem, matlab_output):
 
     def check_dependencies(self):
         # Check Matlab version
-        cmd = sp.check_output(['/usr/local/bin/matlab', '-nodisplay -nosplash -nodesktop -r', '"disp(version); quit"'])
-        m_version = ' '.join(cmd.decode('utf-8').split()[-1:]).replace('(','').replace(')','')
+        cmd = sp.check_output(['/usr/local/bin/matlab', '-nodisplay -nosplash -nodesktop -batch', '"disp(version); quit"'])
+        m_version = cmd.decode('utf-8').replace('MATLAB is selecting SOFTWARE OPENGL rendering.','').replace('\n','')
```

Comment:
The reverse match was a unstable indeed, but it's alright long as the Matlab version doesn't change (it's a manual process). 
Matlab could decide to slightly alter that first line in the printout too, so I'm not seeing this version as being much more robust.
One could change the indexes from `[-1:]` to `[-2:-1]` or to `[7:8]`, but it's equally unstable.

Reply:
@mihai-sysbio I see your point about stability here. When taking a look to MATLAB's version and release history I can see that their formats are quite heterogeneous ([here in the "Release history" section](https://en.wikipedia.org/wiki/MATLAB)). Since 2016 they have been consistent, at least in the `release` part of the version string (`R + year + (a | b)`).  

Taking this into account, commit  0ad6bdf suggests the use of regular expressions applied on the utf-8 decoded cmd string in order to retrieve the release ID. In order to provide a bit more of stability, commit e355414 flexibilizes the last part of the `release` regular expression to match, as cases such as "R2009bSP1" may arise in MATLAB releases. The parsed version result can be seen [here](https://github.com/SysBioChalmers/ecModels/runs/1942343499?check_suite_focus=true#step:3:7). I think that this is a bit more robust and then we don't need to care about potential changes in the "MATLAB welcome string", as pointed out in your comment.